core/common/assets/js/common.js

```diff
@@ -7,6 +7,7 @@ import Connect from 'elementor-common-modules/connect/assets/js/connect';
 import WordpressComponent from './components/wordpress/component';
 import EventsDispatcherComponent from 'elementor-common-modules/event-tracker/assets/js/data/component';
 import Events from 'elementor-common-modules/event-tracker/assets/js/events';
+import Notifications from 'elementor-editor-utils/notifications';
```

Comment:
Maybe you should move it to the global utils folder?
I do understand it might be a problem with BC but it might be the right way to do it.

Reply:
I was thinking about that, but was not sure what's the right thing to do, so left it there.
But as we both thought the same thing - moving.

Reply:
Yes, but still if you move it, think about BC.